src/main/views/webContentEvents.ts

```diff
@@ -106,17 +110,17 @@ export class WebContentsEventManager {
     };
 
     denyNewWindow = (details: Electron.HandlerDetails): {action: 'deny' | 'allow'} => {
-        log.warn(`Prevented popup window to open a new window to ${details.url}.`);
+        this.log().warn(`Prevented popup window to open a new window to ${details.url}.`);
```

Comment:
If I read the code right, this will log as `[WebContentsEventManager] [undefined] Prevented popup...`. The undefined part is intended?

Reply:
Ahh you're right it will, I'll fix that.

Reply:
Yeah that should be fixed. I'll fix it.